code/modules/cm_marines/smartgun_mount.dm

```diff
@@ -433,6 +433,7 @@
 	if(!burst_fire && target && !last_fired)
 		fire_shot()
 
+	burst_fire = 0
```

Comment:
```suggestion
	burst_fire = FALSE
```
To know why, check `No magic numbers or strings` in our contribution guidelines.